src/snovault/elasticsearch/searches/interfaces.py

```diff
@@ -55,6 +55,7 @@
 NOT_FLAG = '!'
 NOTIFICATION = 'notification'
 OBJECT_FRAME = 'object'
+OPEN_ON_LOAD = 'openOnLoad'
```

Comment:
Should this be `open_on_load` instead of `openOnLoad`? Some companies’ JSON does use camel case (Google I believe) and some use snake. But I think this would be the first JSON key in encoded that uses camel case.